parser/CMakeLists.txt

```diff
@@ -14,15 +14,17 @@ add_executable(CodeCompass_parser
 set_target_properties(CodeCompass_parser
   PROPERTIES ENABLE_EXPORTS 1)
 
-target_link_libraries(CodeCompass_parser
-  util
-  model
+find_boost_library(
```

Comment:
I know originally they weren't in a good order, but in case changing the order does **not** actually break the install, we should order these things alphabetically.

Reply:
It's OK to order them alphabetically, updated accordingly.